packages/account/src/Styles/account.scss

```diff
@@ -1434,6 +1452,15 @@ $MIN_HEIGHT_FLOATING: calc(
                 }
             }
         }
+        .name {
+            .name_content {
```

Comment:
ditto, but if u change to `name-content` or `name__content`, you can use 
`&-content` or `&__content` respectively for the above instead of the full `className`.

Reply:
Correct, in fact, i just followed the existing className which is `last_used_content`, in the same file. nvm i will update both to follow our naming convention, thanks